src/utils/supabase/client.test.ts

```
@@ -0,0 +1,1085 @@
+        testKey,
+        expect.objectContaining({
+          auth: expect.objectContaining({
+            persistSession: expect.any(Boolean),
+            autoRefreshToken: expect.any(Boolean),
+          }),
+        }),
+      )
+    })
+
+    it('should throw error when SUPABASE_URL is missing', () => {
+      delete process.env.NEXT_PUBLIC_SUPABASE_URL
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)